wfe/web-front-end.go

```diff
@@ -278,6 +293,7 @@ func (wfe *WebFrontEndImpl) Challenge(authz core.Authorization, response http.Re
 			sendError(response, "Failed to marshal authz", http.StatusInternalServerError)
 			return
 		}
+	  response.Header().Add("Content-Type", "application/json")
```

Comment:
Indent (probably spaces vs tabs). You can fix all indent issues automatically with gofmt.

Also this helps deal with go style:

```
$ cat .vim/ftplugin/go.vim
setlocal noexpandtab
setlocal sw=2
setlocal ts=2
```

Along with, if necessary, in .vimrc:

```
au BufNewFile,BufRead *.go set filetype=go syntax=go
```